src/java/apiview-java-processor/src/main/java/com/azure/tools/apiview/processor/analysers/JavaASTAnalyser.java

```diff
@@ -302,96 +302,103 @@ private void tokeniseMavenPom(Pom mavenPom) {
 
-        indent();
-        mavenPom.getDependencies().stream().collect(Collectors.groupingBy(Dependency::getScope)).forEach((k, v) -> {
-            if ("test".equals(k)) {
-                // we don't care to present test scope dependencies
-                return;
-            }
-            String scope = k.isEmpty() ? "compile" : k;
+            indentedBlock(() -> {
+                mavenPom.getDependencies()
+                    .stream()
+                    .collect(Collectors.groupingBy(Dependency::getScope))
+                    .forEach((k, v) -> {
+                        if ("test".equals(k)) {
+                            // we don't care to present test scope dependencies
+                            return;
+                        }
+                        String scope = k.isEmpty() ? "compile" : k;
 
-            addToken(makeWhitespace());
-            addToken(new Token(COMMENT, "// " + scope + " scope"), NEWLINE);
+                        addToken(makeWhitespace());
+                        addToken(new Token(COMMENT, "// " + scope + " scope"), NEWLINE);
 
-            for (int i = 0; i < v.size(); i++) {
-                Dependency d = v.get(i);
-                addToken(makeWhitespace());
-                String gav = d.getGroupId() + ":" + d.getArtifactId() + ":" + d.getVersion();
-                addToken(new Token(TEXT, gav, gav));
+                        for (int i = 0; i < v.size(); i++) {
+                            Dependency d = v.get(i);
+                            addToken(makeWhitespace());
+                            String gav = d.getGroupId() + ":" + d.getArtifactId() + ":" + d.getVersion();
+                            addToken(new Token(TEXT, gav, gav));
 
-                if (i < v.size() - 1) {
-                    addNewLine();
-                }
-            }
+                            if (i < v.size() - 1) {
+                                addNewLine();
+                            }
+                        }
+
+                        addNewLine();
+                    });
+            });
 
-            addNewLine();
+            addToken(INDENT, new Token(PUNCTUATION, "}"), NEWLINE);
+            addToken(new Token(SKIP_DIFF_END));
+
+            // allowed dependencies (in maven-enforcer)
+            //        if (!mavenPom.getAllowedDependencies().isEmpty()) {
+            //            addToken(INDENT, new Token(KEYWORD, "allowed-dependencies"), SPACE);
+            //            addToken(new Token(PUNCTUATION, "{"), NEWLINE);
+            //            indent();
+            //            mavenPom.getAllowedDependencies().stream().forEach(value -> {
+            //                addToken(INDENT, new Token(TEXT, value, value), NEWLINE);
+            //            });
+            //            unindent();
+            //            addToken(INDENT, new Token(PUNCTUATION, "}"), NEWLINE);
+            //        }
+
+            // close maven
```

Comment:
Do we need to retain this? Seems like this has been commented out from a long time.

Reply:
I'm not really sure, this has been commented out for a while and I'm guessing this is something we just haven't implemented but want to? @JonathanGiles

Reply:
yeah, it migh as well go for now.